### PR TITLE
Version 0.0.2 Release Candidate

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Release Candidate for version 0.2.0.

# Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality
      to not work as expected)
- This change requires a documentation update

# How Has This Been Tested?

Updated unit tests.

## Test Configuration
* Operating system: OS X Big Sur
* Version number: Python 3.10
* Any details about your local setup that are relevant: develop branch of pysat

# Checklist:

- [x] Update the version number
- [x] Verify and update Zenodo information in .zenodo.json
- [x] doc tests pass locally
- [x] activate this branch on readthedocs (see unit tests list)
- [x] Upload the test distribution to the testPyPi server
- [x] Contacted developers and users about the future release
- [x] Performed integration tests with downstream packages (include list of these packages)
- [x] Create a wiki page with the release text: https://github.com/pysat/pysatSpaceWeather/wiki/v0.2.0
- [ ] After approval, confirm all changes from the RC pull are listed in the wiki release text.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Review by jklenzing — APPROVED** [review]

File: pysatSpaceWeather/instruments/methods/f107.py [hunk 1]
```diff
@@ -117,9 +116,12 @@ def combine_f107(standard_inst, forecast_inst, start=None, stop=None):
     Notes
     -----
     Merging prioritizes the standard data, then the 45day data, and finally
-    the forecast data
+    the forecast data.
 
-    Will not attempt to download any missing data, but will load data
+    Will not attempt to download any missing data, but will load data.
```
Comment (rstoneback) [inline pysatSpaceWeather/instruments/methods/f107.py:121]: I don't quite understand this comment. If a user needs to download data where do they go?

File: pysatSpaceWeather/instruments/methods/kp_ap.py [hunk 2]
```diff
@@ -540,9 +532,12 @@ def combine_kp(standard_inst=None, recent_inst=None, forecast_inst=None,
     Note
     ----
     Merging prioritizes the standard data, then the recent data, and finally
-    the forecast data
+    the forecast data.
+
+    Will not attempt to download any missing data, but will load data.
```
Comment (rstoneback) [inline pysatSpaceWeather/instruments/methods/kp_ap.py:537]: Same comment about download.

File: pysatSpaceWeather/instruments/methods/f107.py (hunk 1 above)
Comment (aburrell) [reply pysatSpaceWeather/instruments/methods/f107.py:121]: The user is providing instruments as stated above.  They don't need to load data into those instruments, but the files need to be available.  If the data can't be loaded, it's treated as missing.  If you find the wording confusing, can you suggest an alternative?  It seems clear to me.

File: pysatSpaceWeather/instruments/methods/swpc.py [hunk 3]
```diff
@@ -962,3 +969,43 @@ def list_files(name, tag, inst_id, data_path, format_str=None):
         files.loc[files.index[-1] + pds_offset] = files.values[-1]
 
     return files
+
+
+def find_issue_date(file_txt, date_fmt="%H%M UT %d %b %Y"):
```
Comment (rstoneback) [inline pysatSpaceWeather/instruments/methods/swpc.py:974]: I'd recommend adding tests for this new function.

Comment (aburrell) [reply pysatSpaceWeather/instruments/methods/swpc.py:974]: Added #150 to address this.

**Review by rstoneback — APPROVED** [review]

**Pull request merged**